test/test_helper.rb

```diff
@@ -1,4 +1,5 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'sidekiq/scheduler/notification'
 
+pp require 'yaml'
```

Comment:
Can you remove this? It seems debug message.

Reply:
oh...

Reply:
fixed